static/js/publisher/form/banner.js

```diff
@@ -170,6 +170,8 @@ class Banner extends React.Component {
                 <br />
                 Max resolution: <b>4320 x 1440 pixels</b>
                 <br />
+                Recommended resolution: <b>1920 x 640 pixels</b>
```

Comment:
Maybe it would be worth adding a bit of explanation why it's recommended? That smaller sizes would be pixelated when displayed in full width on store page?